.gitignore

```diff
@@ -8,6 +8,9 @@ generated/
 target/
```

Comment:
Maybe add `docs-target/` here? This is currently getting ignored by git because `docs-target/` only contains a `target/` sub-directory I think.